idtoken/idtoken.go

```diff
@@ -34,6 +34,7 @@ const (
 	unknownCredType credentialsType = iota
 	serviceAccount
 	impersonatedServiceAccount
+	external_account
```

Comment:
nit: can you please change this to `externalAccount`

Reply:
good catch, I totally overlooked it! Fixed now.